src/main/java/de/tarent/entities/ParticipantFeedback.java

```diff
@@ -0,0 +1,30 @@
+package de.tarent.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import io.quarkus.hibernate.orm.panache.PanacheEntity;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.hibernate.validator.constraints.Length;
+
+import javax.persistence.Entity;
+import javax.validation.constraints.NotNull;
+import java.time.OffsetDateTime;
+
+@Entity
+@EqualsAndHashCode
+@ToString
+@JsonIgnoreProperties(value = {"id"}, ignoreUnknown = true)
+public class ParticipantFeedback extends PanacheEntity {
+
+    @Length(max = 255)
+    public String participant_name;
```

Comment:
Since this just came up in https://github.com/tarent/t-learn/pull/109, why is this field in snake case and not in camel case like all others?

Reply:
good point, maybe I should also rename the table to "participant_feedback" and the timestamp field to feedback_time. Just saw that there are columns contactPerson and executionType. I would suggest camel-case for JSON properties and snake-case for table/column names